centreon/src/Core/Macro/Application/Repository/ReadServiceMacroRepositoryInterface.php

```diff
@@ -37,4 +37,13 @@ interface ReadServiceMacroRepositoryInterface
      * @return Macro[]
      */
     public function findByServiceIds(int ...$serviceIds): array;
+
```

Comment:
Add in comment of the interface, that the service and service template are handled by this interface